playbooks/common/openshift-node/config.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - name: Gather and set facts for node hosts
-  hosts: oo_nodes_to_config
+  hosts:
+  - oo_nodes_to_config
```

Comment:
What's the reasoning for turning this into a list?

I haven't seen this used before, and you're only specifying 1 item, so it doesn't look like it needs to be a list.